packages/fxa-auth-db-mysql/db-server/test/backend/remote.js

```diff
@@ -141,176 +141,163 @@ module.exports = function(cfg, makeServer) {
       );
     });
 
-    it('add account, add email, get secondary email, get emails, delete email', () => {
-      var user = fake.newUserDataHex();
-      var secondEmailRecord = user.email;
-      var thirdEmailRecord;
+    it('add account, add email, get secondary email, get emails, delete email', async () => {
```

Comment:
Huh, I thought I left a comment about this test update, but it doesn't seem to be here.

So, this test started failing for me in CI, and I tracked it down to an issue with determinism - this PR seems to somehow perturb the order in which secondary emails are returned by the db, and this test was depending on them being in a specific order.  The only functional change is that I've added a check to put the results in a consistent order, but in the process of debugging this test I asyncified it, hence the churny diff.